docs/UserGuide.md

```diff
@@ -476,12 +481,18 @@ Format: `edit -p PATIENT_INDEX -d TASK_INDEX d/<TASK_DESCRIPTION> | <DATE TIME>
 * Edits the task at the specified `TASK_INDEX` of the patient at the specified `PATIENT_INDEX`.
 * The task index refers to the index number shown in the task list of a patient.
 * `DATE TIME` and `INTERVAL TIME_PERIOD` must follow the criteria defined in notes under command format.
-* If no new `DATE TIME` or `INTERVAL TIME_PERIOD` are provided, then original values will be used.
+* If there is no change for certain parameters from the original values, leave them as empty but use the `|` if the position of that parameter happens to precede the parameter to be edited, if the position of the parameter is after the parameter to be edited then `|` is not required (refer to examples below).
 * If a `INTERVAL TIME_PERIOD` is provided for what was originally a non-recurring task, the edit will transform it into a recurring one based on the given frequency
+* Note that tasks are automatically sorted in chronological order upon being edited, i.e if a task on `25-10-22` is edited to be `30-10-22`, it's new `TASK INDEX` would be based on its chronological order in the patients task list.
 
 Examples:
 * `list` followed by `edit -p 1 -d 1 d/Administer 3ml of example medicine` edits the description of the 1st task of the 1st patient in the patient list to `Administer 3ml of example medicine`, while retaining the original date and time for the task.
-* `find Betsy` followed by `edit -p 2 -d 3 d/| 23-10-22 0800` edits the date and time of the 3rd task of the 2nd patient in results of the `find` command to 23rd October 2022 0800 hours.
+* `find Betsy` followed by `edit -p 2 -d 3 d/ | 23-10-22 0800` edits the date and time of the 3rd task of the 2nd patient in results of the `find` command to 23rd October 2022 0800 hours, while retaining the original description for the task.
```

Comment:
Perhaps it is better to keep the use of pipe characters consistent, like in the medication section. There is no space between `d/` and `|` (even if it is accepted, still good to be consistent).

```suggestion
* `find Betsy` followed by `edit -p 2 -d 3 d/| 23-10-22 0800` edits the date and time of the 3rd task of the 2nd patient in results of the `find` command to 23rd October 2022 0800 hours, while retaining the original description for the task.
```